lib/6to5/transformation/transformers/optional-external-runtime.js

```diff
@@ -0,0 +1,11 @@
+exports.optional = true;
+
+// In theory, it would be more appropriate to do this in `manipulateOptions`,
+// but we need an identifier for the import and we can't get that before the
+// AST is built.
+exports.ast = {
+  enter: function (ast, file) {
+    file.opts.runtime = file.addImport(file.opts.runtime, "to5Runtime").name;
+    file.dynamicImports[file.dynamicImports.length - 1]._noInteropRequire = true;
```

Comment:
You can set `_blockHoist = 3;` on the node to hoist it above **all** other nodes so it'll be before all helpers.